From b7d3a3c727b0f1d14c74d6302a98fb60f0aec1b8 Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Thu, 13 Jun 2024 10:26:33 +0000
Subject: [PATCH 09/24] rdk/snr: klm/cpk/ice_main.c: add extack support to
 ice_fdb_del

kernel commit ca4567f1e6f6 ("rtnetlink: add extack support in fdb del
handlers").

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/cpk/ice_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/klm/cpk/ice_main.c b/klm/cpk/ice_main.c
index af355fb..509f146 100644
--- a/klm/cpk/ice_main.c
+++ b/klm/cpk/ice_main.c
@@ -4239,7 +4239,7 @@ ice_fdb_add(struct ndmsg *ndm, struct nlattr __always_unused *tb[],
 static int
 ice_fdb_del(struct ndmsg *ndm, __always_unused struct nlattr *tb[],
 	    struct net_device *dev, const unsigned char *addr,
-	    __always_unused u16 vid)
+	    __always_unused u16 vid, struct netlink_ext_ack *extack)
 #else
 static int
 ice_fdb_del(struct ndmsg *ndm, __always_unused struct nlattr *tb[],
-- 
2.39.3

